libs/interpret-vision/src/lib/VisionExplanationDashboard/VisionExplanationDashboard.styles.ts

```diff
@@ -37,18 +43,18 @@ export const visionExplanationDashboardStyles: () => IProcessedStyleSet<IDataset
         width: "50%"
       },
       imageListContainer: {
-        border: "1px solid #D2D4D6",
+        border: `1px solid ${theme.semanticColors.disabledBorder}`,
         height: "100%",
         overflow: "scroll"
       },
-      line: {
-        borderTop: "1px solid #EDEBE9"
-      },
       mainContainer: {
         alignItems: "center",
         justifyContent: "space-between",
         width: "100%"
       },
+      mainImageContainer: {
+        height: "650px"
+      },
```

Comment:
Please check on zoom-in/out if this renders as expected. As We are working on Reflow tasks, it is better to have reflow on newly added code.

Reply:
It works for me! But I can look into the reflow thing if needed